python/ccf/bundle_js_app.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+import argparse
+import glob
+import json
+import os
+import shutil
+import tempfile
+
+
+def read_modules(modules_path):
+    modules = []
+    for path in glob.glob(f"{modules_path}/**/*", recursive=True):
+        if not os.path.isfile(path):
+            continue
+        rel_module_name = os.path.relpath(path, modules_path)
```

Comment:
os.posixpath (as opposed to os.path) will do the right thing and won't need line 18